tests/test_kernels.py

```diff
@@ -137,15 +132,14 @@ def test_arccosine_non_implemented_order(order):
         gpflow.kernels.ArcCosine(order=order)
 
 
-@pytest.mark.parametrize('ard', [True, False])
```

Comment:
I would keep this in - maybe as
```python
@pytest.mark.parametrize('weight_variances', [
    1.,  # ard = False
    [1., 1., 1.],  # ard = True
])
```
(and then take `weight_variances` out of the following parametrization)
(assumes input_dim=3)

Reply:
or did this already get tested elsewhere ? :thinking:

Reply:
Actually - this is now sufficiently tested in `test_arccosine_kern_err` - this test can be removed